templates/lp-page/lp-page.css

```diff
@@ -0,0 +1,3 @@
+h2{
+    text-align: center;
```

Comment:
Instead of setting all H2 for this template as centered, I think we need a global way to set headings centered ad-hoc. I created this function for Sling, see https://github.com/aemsites/sling/pull/268/files

Because these headlines are already using a bold font, there's no visual change to the font.

Reply:
ok..lemme do this way...